tdrs-frontend/src/axios-instance.js

```diff
@@ -1,9 +1,16 @@
 import axios from 'axios'
+import tough from 'tough-cookie'
+import { wrapper } from 'axios-cookiejar-support'
+
+wrapper(axios)
```

Comment:
I don't think we need any of this file with the lower level fix I did in auth.js